README.md

```diff
@@ -65,39 +64,56 @@ MessagePack library. For the greatest benefit though, we recommend using
 **Define** your message schemas using standard Python type annotations.
 
 ```python
->>> import msgspec
+>> > import msgspec
+
+>> >
 
->>> class User(msgspec.Struct):
-...     """A new type describing a User"""
-...     name: str
-...     groups: set[str] = set()
-...     email: str | None = None
+class User(msgspec.Struct):
+    ...
+
+
+"""A new type describing a User"""
+...
+name: str
+...
+groups: set[str] = set()
+...
+email: str | None = None
 ```
 
 **Encode** messages as JSON, or one of the many other supported protocols.
 
 ```python
->>> alice = User("alice", groups={"admin", "engineering"})
+>> > alice = User("alice", groups={"admin", "engineering"})
 
->>> alice
+>> > alice
 User(name='alice', groups={"admin", "engineering"}, email=None)
 
->>> msg = msgspec.json.encode(alice)
+>> > msg = msgspec.json.encode(alice)
 
->>> msg
+>> > msg
 b'{"name":"alice","groups":["admin","engineering"],"email":null}'
 ```
 
 **Decode** messages back into Python objects, with optional schema validation.
 
 ```python
->>> msgspec.json.decode(msg, type=User)
+>> > msgspec.json.decode(msg, type=User)
 User(name='alice', groups={"admin", "engineering"}, email=None)
 
->>> msgspec.json.decode(b'{"name":"bob","groups":[123]}', type=User)
-Traceback (most recent call last):
-  File "<stdin>", line 1, in <module>
-msgspec.ValidationError: Expected `str`, got `int` - at `$.groups[0]`
+>> > msgspec.json.decode(b'{"name":"bob","groups":[123]}', type=User)
+Traceback(most
+recent
+call
+last):
+File
+"<stdin>", line
+1, in < module >
+msgspec.ValidationError: Expected
+`str`, got
+`int` - at
+`$.groups[0]
+`
```

Comment:
Looks like your editor auto-formatted things? All changes here should be reverted.